### DISCUSSION TITLE
Diferença entre entre Exception ex.Message e o throw new exception

### DISCUSSION THREAD

**Discussion by pedro-mileipp** [balta-io/forum#138] [Tira Dúvidas] [answered]
- Boa tarde, qual seria a diferença das duas formas, na pergunta exemplifiquei com tipos genéricos, mas caso eu defina corretamente o tipo do erro tanto faz qual usar? Abraços!

**Comment by andrebaltieri** [comment, marked as answer]
Boa noite, @pedro-mileipp, como vai? Poderia ilustrar com algum código a sua pergunta? Não entendi muito bem.
